bootstrap/src/main/config/mediaserver.xml

```diff
@@ -47,7 +47,10 @@
 	<resources>
 		<localConnection poolSize="100" />
 		<remoteConnection poolSize="50" />
-		<player poolSize="50" />
+		<player>
+			<poolSize>50</poolSize>
+			<cacheSize>100</cacheSize>
```

Comment:
@hrosa it's cache size in megabytes. It does not depend on traffic load, but on the size of media files, you're going to play back to a user.